mmhuman3d/data/data_converters/penn_action.py

```diff
@@ -77,7 +77,7 @@ def convert(self, dataset_path: str, out_path: str) -> dict:
                 ]
                 bbox_xywh = self._bbox_expand(bbox_xyxy, scale_factor=1.2)
 
-                image_path_.append(img_path)
+                image_path_.append(img_path.replace(dataset_path + '/', ''))
```

Comment:
Please provide some comments to explain this step.

Reply:
added in f7f4aa9